### DISCUSSION TITLE
KRS-TV

### DISCUSSION THREAD

**Discussion by las-pen** [iptv-org/community#298] [Show and tell]
### Channel ID

KRSTV.fi

### Stream URL

http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UCbS4WD5zGN0EJsaGaZIKeoQ/live

### Notes

iptv-org/iptv#8959 But with proper Channel ID

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UCbS4WD5zGN0EJsaGaZIKeoQ/live

> No streams found.

**Comment by AntiPontifex** [comment]
Dead link: please close.
